bandit/src/query-lambda/queryBigQueryData.ts

```diff
@@ -0,0 +1,119 @@
+import {format, subDays} from "date-fns";
```

Comment:
Other filenames in this repo are hyphenated.
Also, the function in this file just builds the query, whereas I think the filename suggests it runs the query